src/main/resources/swagger/api-docs.yaml

```diff
@@ -314,6 +314,13 @@ definitions:
       - googleProject
       - googleServiceAccount
       - googleBucket
+      - numberOfWorkers
```

Comment:
If we're picking defaults for all these then they shouldn't be in the `required:` list.